google/cloud/aiplatform/training_jobs.py

```diff
@@ -2575,7 +2585,7 @@ def __init__(
                 "minimize-rmse" (default) - Minimize root-mean-squared error (RMSE).
                 "minimize-mae" - Minimize mean-absolute error (MAE).
                 "minimize-rmsle" - Minimize root-mean-squared log error (RMSLE).
-            column_transformations (Optional[Union[Dict, List[Dict]]]):
+            column_specs (Dict[str, str]):
```

Comment:
It seems confusing to have the same docstring for for both `column_specs` and `column_transformations`. The docstring for `column_specs` should be updated to describe the expected format for `column_spec` along with a reference to accepted transformation values.